main.go

```diff
@@ -53,6 +54,9 @@ var (
 	tlsCertFile string
 	tlsKeyFile  string
 
+	tlsMinVersion     string
+	tlsMinVersionCode uint16
```

Comment:
Keep two values to be easier to compare if it is set or not this parameter.

Reply:
Umm, why not using a tls value directly here? We would also have the members for further uses.
```suggestion
	tlsMinVersion     *tlsMinVersionValue
```

It could implement an `IsZero() bool` method to make it easy to check if it is set or not :slightly_smiling_face: